include/odp/api/spec/traffic_mngr.h

```diff
@@ -385,6 +440,10 @@ typedef struct {
 	 * the parameters of the threshold profile of any TM node or TM queue.
 	 */
 	odp_bool_t dynamic_threshold_update;
+
+	/** Statistics counters capabilities */
+	odp_tm_stats_capability_t stats;
```

Comment:
For node stats, capabilities should be per level. So Maybe name it explicitly as odp_tm_queue_stats_capability ?

Reply:
Makes sense if we don't require additional TM level statistics capabilities. Do you know if this is the case?

Reply:
Fixed in V3

Reply:
Missed to respond to your comment. If you mean other global TM statistics apart from queue stats, then I think yes, atleast our HW doesn't support such stats which are global and not queue related.